src/database/queries.ts

```diff
@@ -100,3 +106,49 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Create a new "vendor" document and at it to the vendors collection. 
+ */
+ export async function createVendor(email: string, name: string) {
```

Comment:
when creating a new vendor, the UUID doesn't get assigned – rather than logging `docref.id`, assign this to the vendor